agent/fw_wordpress.c

```diff
@@ -578,6 +580,65 @@ NR_PHP_WRAPPER(nr_wordpress_apply_filters) {
 }
 NR_PHP_WRAPPER_END
 
+#if ZEND_MODULE_API_NO >= ZEND_7_4_X_API_NO
+/*
+ * Wrap the wordpress function add_filter
+ *
+ * function add_filter( $hook_name, $callback, $priority = 10, $accepted_args = 1 )
+ *
+ * @param string   $hook_name     The name of the filter to add the callback to.
+* @param callable $callback      The callback to be run when the filter is applied.
+* @param int      $priority      Optional. Used to specify the order in which the functions
+*                                associated with a particular filter are executed.
+*                                Lower numbers correspond with earlier execution,
+*                                and functions with the same priority are executed
+*                                in the order in which they were added to the filter. Default 10.
+* @param int      $accepted_args Optional. The number of arguments the function accepts. Default 1.
+* @return true Always returns true.
+ */
+
+NR_PHP_WRAPPER(nr_wordpress_add_filter) {
+  /* Wordpress's add_action() is just a wrapper around add_filter(),
+   * so we only need to instrument this function */
+  NR_UNUSED_SPECIALFN;
+  (void)wraprec;
+  const char* filename = NULL;
+  bool wrap_hook = true;
+
+  NR_PHP_WRAPPER_REQUIRE_FRAMEWORK(NR_FW_WORDPRESS);
+
+  /*
+   * We only want to hook the function being called if this is a WordPress
+   * function, we're instrumenting hooks, and WordPress is currently executing
+   * hooks (denoted by the wordpress_tag being set).
+   */
+  if ((NR_FW_WORDPRESS != NRPRG(current_framework))
+      || (0 == NRINI(wordpress_hooks))) {
```

Comment:
I think this check needs to be added in `oapi` branch too.

Reply:
This will be handled when `oapi` branch is synchronized with these changes after they land in `dev`.